conf/supervisord-dev.conf

```diff
@@ -10,8 +10,15 @@ stderr_events_enabled=true
 stopsignal = KILL
 stopasgroup = true
 
-[program:beat]
-command=celery -b %(ENV_BROKER_URL)s -A h_periodic.beat beat
+[program:h-beat]
+command=celery -b %(ENV_BROKER_URL)s -A h_periodic.h_beat beat --pidfile=celerybeat.pid
```

Comment:
`h-celerybeat.pid` maybe

Reply:
Ditto for above. I didn't want to mash existing things, but this would be nicer.

Reply:
Done